toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt

```diff
@@ -174,6 +175,12 @@ public fun MapView(
             it.backgroundGrid = backgroundGrid
             it.isAttributionBarVisible = isAttributionBarVisible
             it.setTimeExtent(timeExtent)
+            if (mapView.graphicsOverlays != graphicsOverlays) {
```

Comment:
Better to use `it` for consistency:
```suggestion
            if (it.graphicsOverlays != graphicsOverlays) {
```